### ISSUE TITLE
mlproj deploy shows error

### ISSUE THREAD

**Issue by eurochriskelly** [fgeorges/mlproj-core#1]
When running using default env it shows an error. Advice would be better.

**Comment by fgeorges** [comment]
Can you give a repro?  An example project would be great, but at least the command you used (and under which condition it fails)

**Comment by eurochriskelly** [comment]
**Repro:**

```
$ mkdir ooo
$ cd ooo
$ mlproj new # choose defaults
$ mlproj setup
$ mlproj delpoy # unhandled error

λ mlproj deploy
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: Server has no modules database: ooo
    at target (C:\Users\ck-tahzoo\AppData\Roaming\npm\node_modules\mlproj\node_modules\mlproj-core\src\commands.js:239:31)
    at DeployCommand.execute (C:\Users\ck-tahzoo\AppData\Roaming\npm\node_modules\mlproj\node_modules\mlproj-core\src\commands.js:304:24)
    at XProject.execute (C:\Users\ck-tahzoo\AppData\Roaming\npm\node_modules\mlproj\node_modules\mlproj-core\src\space.js:164:17)
    at platform.project.project (C:\Users\ck-tahzoo\AppData\Roaming\npm\node_modules\mlproj\src\tui.js:149:17)
    at prj.load (C:\Users\ck-tahzoo\AppData\Roaming\npm\node_modules\mlproj\node_modules\mlproj-core\src\space.js:39:17)
    at platform.xml (C:\Users\ck-tahzoo\AppData\Roaming\npm\node_modules\mlproj\node_modules\mlproj-core\src\space.js:228:17)
    at parser.parseString (C:\Users\ck-tahzoo\AppData\Roaming\npm\node_modules\mlproj\src\node.js:117:17)
    at Parser.<anonymous> (C:\Users\ck-tahzoo\AppData\Roaming\npm\node_modules\mlproj\node_modules\xml2js\lib\xml2js.js:489:18)

    at emitOne (events.js:96:13)
    at Parser.emit (events.js:188:7)
```
Instead of the big error, I would expect "Hey, you can't deploy when using (filesystem)! Check your env!"

**Comment by fgeorges** [comment]
I have implemented some basic error reporting.  It is published in mlproj-core 0.10.2 and mlproj 0.32.0.

Can you upgrade mlproj to the latest version and try if it suits you?

**Comment by eurochriskelly** [comment]
Works. Thanks!
